setup.py

```diff
@@ -27,6 +27,7 @@
         'monitoring' : ['psutil', 'sqlalchemy', 'sqlalchemy_utils'],
         'aws' : ['boto3'],
         'kubernetes' : ['kubernetes'],
+        'globus_ssh' : ['globus-ssh>=0.5'],
```

Comment:
I'm not sure what is happening here: pip in my two dev envs (3.5 and 3.7) cannot find globus-ssh, and although google gives a link to it on pypy, that link resolves to a 404 for me at the moment: https://pypi.org/project/globus-ssh/

That means I can't install the required dependencies.

Reply:
Updated to use the 'oauth-ssh' package.